src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -1666,20 +1666,6 @@ public void run() {
 	public void dropItemAtLocation(Block b, ItemStack is) {
 		dropItemAtLocation(b.getLocation(), is);
 	}
-  
-  private void damageTool(ItemStack is) {
```

Comment:
I have a feeling this was used by a piece of code commented out or removed by Maxopoly's purge, some of which I need to go back and re-add, iirc.

So many things todo. 

In any case for the moment, this case remain removed. If I need it, I'll find it. This comment is mostly for me ;)